packages/common/index.ts

```diff
@@ -230,12 +230,15 @@ export enum OpenQuestionStatus {
 export enum ClosedQuestionStatus {
   Resolved = "Resolved",
   Deferred = "Deferred",
-  NoShow = "No Show",
+  NoShow = "NoShow",
   Deleted = "Deleted",
 }
 
 // Ticket Status - Represents a given status of as student's ticket
-export const QuestionStatus = {
+export type QuestionStatus = keyof typeof QuestionStatusKeys;
+// IMPORTANT: whenever importing QuestionStatusKeys to use for any reason, import it from
+// @template/common/index. for some reason, importing from @template/common results in undefined.
```

Comment:
this bug is absolutely disgusting. took me an hour before I was able to find out this issue exists.

Reply:
Ouch that looks like a tough one. Nice work hunting down the issue!